pkg/api/vlabs/types.go

```diff
@@ -125,6 +125,7 @@ type OrchestratorProfile struct {
 // Kubernetes specific configuration
 type KubernetesConfig struct {
 	KubernetesImageBase string `json:"kubernetesImageBase,omitempty"`
+	NetworkPolicy       string `json:"networkPolicy,omitempty"`
```

Comment:
NetworkConfig will not be limited to KubernetesConfig. That's the point of adding a type (so it can be reused by other orchestrator's config when the need arises).

>>Calico is not really used for any networking.
But its still used for managing iptables so it is a network config. It makes sense for all network policies and other network settings to be inside network config so that the object structure aka API model is setup correctly from the get go and won't require revisiting.